mycore-ifs/src/main/java/org/mycore/datamodel/niofs/ifs2/MCRFileSystemUtils.java

```diff
@@ -309,4 +317,30 @@ private static Optional<MCRDirectory> doResolveParent(MCRPath parent) {
             .map(MCRDirectory.class::cast);
     }
 
+    static Collection<String> getObjectBaseIds() throws IOException {
+        final Path baseDir = Paths.get(getBaseDir());
+        if (!Files.isDirectory(baseDir)) {
+            return List.of();
+        }
+        try (Stream<Path> dirEntries = Files.list(baseDir)) {
+            return dirEntries
+                .filter(Files::isDirectory)
+                .sorted(Comparator.comparing(Path::getFileName))
+                .flatMap(dir -> {
+                    try (Stream<Path> subDirEntries = Files.list(baseDir)) {
```

Comment:
Shouldn't it be `Stream<Path> subDirEntries = Files.list(dir)` at this point? Otherwise, the variable `dir` is never used.

Reply:
Also, assuming the above assumtion is true, better and less confusing naming would be 

- `Stream<Path> baseDirEntries = Files.list(baseDir)` (renaming `dirEntries` to `baseDirEntries`)
-  `Stream<Path> subDirEntries = Files.list(subDir)` (ranaming `dir` to `subdir`)